### PR TITLE
Pstweak

### PR DESCRIPTION
**Pull request by sirspinach**
`quilt ps` now looks like this:
``` % quilt ps  
MACHINES
ID    ROLE      PROVIDER    REGION       SIZE        PUBLIC IP        CONNECTED
11    Master    Amazon      us-west-1    m4.large    52.53.189.143    true
12    Worker    Amazon      us-west-1    m4.large    54.153.119.70    true
13    Worker    Amazon      us-west-1    m4.large    52.53.215.96     true
14    Worker    Amazon      us-west-1    m4.large    52.53.223.207    true

CONTAINERS
ID    MACHINE       CONTAINER      LABELS      PUBLIC IP
2     Machine-12    nginx:1.10     web_tier    54.153.119.70:80
                                               
1     Machine-14    nginx:1.10     web_tier    52.53.223.207:80
```

### CODE REVIEW

File: quiltctl/command/command_test.go [hunk 1]
```diff
@@ -47,8 +47,9 @@ func TestMachineOutput(t *testing.T) {
 	* errors easier to debug. */
```
Comment (ejj) [inline quiltctl/command/command_test.go]: Somewhat of a nit, but could you rewrite the commit message in prose?

Comment (sirspinach) [inline quiltctl/command/command_test.go]: my first thought: "I wrote the commit message in verse?" haha

Comment (sirspinach) [inline quiltctl/command/command_test.go]: done

**Comment by sirspinach** [comment]
Updated spark test to use quilt client instead of parsing the output of `quilt containers`.

**Pull request merged**